guides/plugins/plugins/administration/using-custom-fields.md

```diff
@@ -0,0 +1,98 @@
+begin with. Here you can learn how to add your custom fields:
+
+{% page-ref page="./../framework/custom-field/add-custom-field.md" %}
+
+## Using custom fields in your module
+
+In Shopware, we provide an own component called `sw-custom-field-set-renderer` for your template, being tailored 
+specifically to display custom field sets. 
+
+As a consequence, you're able to use this component to display your custom fields. See here:
+
+{% code title="<plugin-root>/src/Resources/app/administration/app/src/component/swag-basic-example/swag-basic-example.html.twig" %}
+```html
+<sw-card title="Custom fields">
+    <sw-custom-field-set-renderer
+        :entity="customEntity"
+        showCustomFieldSetSelection
+        :sets="sets">
+    </sw-custom-field-set-renderer>
+</sw-card>
+```
+{% endcode %}
+
+For further details on the `sw-custom-field-set-renderer` component, feel free to refer to its page in our component 
+library:
+
+<!-- markdown-link-check-disable -->
+{% embed url="https://component-library.shopware.com/components/sw-custom-field-set-renderer" %}
+<!-- markdown-link-check-enable -->
+
+The next step is loading your custom fields. First things first, create a variable for your custom fields in `data`:
+
+{% code title="<plugin-root>/src/Resources/app/administration/app/src/component/swag-basic-example/swag-basic-example.html.twig" %}
+```javascript
+    data() {
+        return {
+            ...
+            yourCustomFields: null
+        };
+    }
+```
+{% endcode %}
+
+Afterwards, you can start to integrate the custom field data into your component. Therefore, you need to create a 
+`customFieldSetRepository` first as `computed` property. In this context, it may come in handy to already set the 
+`customFieldSetCriteria`. Both steps can be seen in the example below:
+
+{% code title="<plugin-root>/src/Resources/app/administration/app/src/component/swag-basic-example/swag-basic-example.html.twig" %}
+```javascript
+computed: {
+    // Using the repository to work with customFields
+    customFieldSetRepository() {
+        return this.repositoryFactory.create('custom_field_set');
+    },
+        
+    // sets the criteria used for your custom field set
+    customFieldSetCriteria() {
+        const criteria = new Criteria();
+
+        // restrict the customFieldSets to be associated with products
+        criteria.addFilter(Criteria.equals('relations.entityName', 'product'));
+
+        // sort the customFields based on customFieldPosition
+        criteria
+            .getAssociation('customFields')
+            .addSorting(Criteria.sort('config.customFieldPosition', 'ASC', true));
+        
+        return criteria;
+    }
+}
+```
+{% endcode %}
+
+Now you can access your custom fields, e.g. within a `method`. In order to achieve that, you can use the 
+`search` method as you're used to working with repositories:
+
+```javascript
```

Comment:
code block missing here ?

Reply:
No, I actually did this by purpose as it's only a code snippet. The reader can use it in his methods the way he needs, can't he?

Reply:
Okay. Its fine for me :-)